### DISCUSSION TITLE
스마트스토어 내 '수량별 부분 클레임' 적용 예정일 문의 드립니다.

### DISCUSSION THREAD

**Discussion by hjseol** [commerce-api-naver/commerce-solutions#30] [묻고 답하기]
안녕하세요.
이지어드민 개발팀 입니다.

기존에 공지해주셨던 아래 내용 확인 하였습니다.
https://github.com/commerce-api-naver/commerce-solutions/discussions/5 

* 스마트스토어: 적용일 이후 수량 클레임 확대 적용 시 반영 예정 (일정은 차후 별도 공지 예정)
-> 일정은 차후 별도로 공지해주신다고 하였으나, 혹시 대략적인 시점이라도 알 수 있을까요?

확인 부탁 드립니다.
감사합니다~

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

현재 네이버페이 수량 클레임은 네이버쇼핑 장보기 일부 주문건에서만 제공하고 있습니다.
수량 클레임은 최소 클레임 단위의 변경인 만큼 판매자나 커머스솔루션 개발사에서도 매우 큰 수정이 필요한 대응 작업임을 감안하여 충분한 대응 기간을 드린 뒤 확대 적용을 구상하고 있습니다.

일반 스마트스토어 판매자 주문건의 적용 시점은 아직까진 구체적으로 결정된 내용이 없습니다. 적어도 남아있는 2024년 연내 기간에는 확대 적용하려는 계획은 갖고있지 않습니다.
판매자 분들과 커머스솔루션 개발사에 의견을 들으며 차후 구체적인 적용 시점을 결정하여 알려드리도록 하겠습니다.
